src/main/java/edu/uci/ics/crawler4j/crawler/CrawlConfig.java

```diff
@@ -134,6 +134,21 @@
     private boolean shutdownOnEmptyQueue = true;
 
     /**
+     * Wait this long before checking the status of the worker threads
+     */
+    private int threadMonitoringDelay = 10;
+
+    /**
+     * Wait this long in seconds to verify the craweler threads are finished working.
+     */
+    private int threadShutdownDelay = 10;
+
+    /**
+     * Wait this long in seconds before launching cleanup.
+     */
+    private int cleanupDelay = 10;
```

Comment:
It is a good idea to put units of measurements inside of variables names instead of using comments.
threadMonitoringDelayInSeconds , threadShutdownDelayInSeconds, cleanupDelayInSeconds gives more understanding.